### PR TITLE
Fix ignition config template when multiple ssh keys are passed

### PR DESCRIPTION
**Pull request by alvaroaleman**
Fixes the indent when multiple ssh keys are passed. Sample generated
configmap:

```
$ k get configmap ignition-config-99-worker-ssh -ojson|jq .data.config -r
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  name: 99-worker-ssh
  labels:
    machineconfiguration.openshift.io/role: worker
spec:
  config:
    ignition:
      version: 2.2.0
    passwd:
      users:
      - name: core

        sshAuthorizedKeys:
        - |-
          ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDHBgwMTqA8AFS3UhHUXkYZ7pkWnGdFqB18uz/5Ai0mT0qgN1aY4kiGruI2uDgiTu1s3ypBrPr8Zg8Vqurrvr4p+BM0lfWQyslG+Q/UecYy0g6qAs9NriT2iWLlCUb9oFJ8Um30Lmj5I00DjnxjEDAY+nTijJvOK0fd+0HYwKmAWYLEDrGNXxW1EVoKmYoO0+Xp3p+K2xJ/7KIiKMyLVcYs5qbNHkTgywjsIW7DLhEkLIRXeubjwV1YHgsxg+8v4jCHYCrabRtG/gMLCQ7+CFST6IxwTKQB1u7U+2zjRpafOLJf/DHCGiQavqVDEZ36xYYvs67hlf3Fs3fwK4VJ0cuh Alvaro Aleman <wouldntyouliketoknow@gmail.com>
          ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC7xaGqJaFd51jCl+MjZzgH1WfgKbNmn+AbvRXOabeNYNRTZiRNcFlWHQPxL/fFWiJ5rDkyTRm6dI49TflU5lMSOcKwoO0sZlMbrDrUeDf2cy/7KffpAto+Te8vB4udAERMJHY89v9/RF6GgMLpW+lbIT3Gyj+MbIF8aAz0vt6VJA8Ptwq2SlxWSPLbxoe5nNP1JaOubG4Arm6t75smJ+wvexV8d9duvFWig2MW5lMTAa6QpSAp6Gd03dWSUiH5++dk3vlNMR9hZMv7/DWqyauGi0MYtuywQqVWr3YMQve72VJTo/qVhvfFylKEFTKA0h5Cl3ziL0DbgM/RDsUqaLynB7b6jAJkhXd02wv6+IkHly02SEnLHGJs50uK7J7GdAWWbKfRByVGg5kP5DwiTEln357ukT7OH8Ys6PNd0Lzzy/oA4Gv+uDzI1RMMBsTcv3SwASuht+EZzQ5hoSCkM6QoEtpruSCEdCtvTEq9idcrVijKbYURtrDdH5WAN9ZYUF13s94870srbG3uavvT2G1IcWjBjiVVoJM8cifYnTHllHX/oPw9iZxhjlrC5Uc+dgRhnpoRYMar30Kg/No1GYj2EPEZgvHVde6KqActTFnD0K5xJEAUzKutu7TDUePm+MYREt4HMeT4LxsVUar9Aak5pgmUKLqKHLY8NeQxWtKMbQ== alvaro@localhost.localdomain
```

Fixes https://github.com/openshift/hypershift/issues/476
/cc @sjenning @enxebre

### CODE REVIEW

File: control-plane-operator/controllers/hostedcontrolplane/render/manifests_test.go [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+			ctx := newClusterManifestContext(nil, map[string]string{"release": "1.2.3"}, tc.params, nil, nil, nil)
+			ctx.ignitionConfigs()
+			for name, value := range ctx.manifests {
+				CompareWithFixture(t, value, WithExtension(name))
+			}
+		})
+	}
+}
+
+// CompareWithFixture will compare output with a test fixture and allows to automatically update them
```
Comment (alvaroaleman) [inline control-plane-operator/controllers/hostedcontrolplane/render/manifests_test.go:47]: This and everything below is pretty much a verbatim copy of https://github.com/openshift/ci-tools/blob/b712c7aba448163bb8f350db9c0311bc27956bad/pkg/testhelper/testhelper.go

Do we have something similiar? If not, how do ppl feel about adding it?

Using fixtures to quickly be able to check as a human if a rendered template looks as expected is IMHO very useful and I love that they can be generated as opposed to when they are embedded as a string into code.

Comment (enxebre) [reply control-plane-operator/controllers/hostedcontrolplane/render/manifests_test.go:47]: I have personally no objection other than maintaining that new code which seems unlikely to change so sgmt.

**Comment by enxebre** [comment]
ci/prow/verify  complains have you may be forgotten to push one of the generated files?

**Comment by sjenning** [comment]
just for good measure
/test e2e-aws-pooled

**Comment by sjenning** [comment]
I tested this locally and it works
/test e2e-aws

**Pull request merged**